packages/next/cli/next-dev.ts

```diff
@@ -82,8 +82,8 @@ const nextDev: cliCommand = (argv) => {
   // we allow the server to use a random port while testing
   // instead of attempting to find a random port and then hope
   // it doesn't become occupied before we leverage it
-  if (process.env.__NEXT_RAND_PORT) {
-    port = 0
+  if (process.env.__NEXT_FORCED_PORT) {
+    port = parseInt(process.env.__NEXT_FORCED_PORT, 10) || 0
   }
```

Comment:
Is this meant to be a separate PR? I dont see how its related to the bug described in the PR desc